dot/state/storage.go

```diff
@@ -167,7 +167,7 @@ func (s *StorageState) loadTrie(root *common.Hash) (*trie.Trie, error) {
 
 	tr, err := s.LoadFromDB(*root)
 	if err != nil {
-		return nil, errTrieDoesNotExist(*root)
+		return nil, fmt.Errorf("%s, %w", err, errTrieDoesNotExist(*root))
```

Comment:
A tiny bit out of scope, but let's just remove `errTrieDoesNotExist` please, it makes everything just more confusing. It's also only used in one other place, remove it there too :pray: 

```suggestion
		return nil, fmt.Errorf("trie does not exist at root %s: %w", *root, err)
```